rpm/xcache.spec

```diff
@@ -40,6 +40,31 @@ Obsoletes: stashcache-daemon < 1.0.0
 %postun
 %systemd_postun_with_restart xcache-reporter.service xcache-reporter.timer xrootd-renew-proxy.service xrootd-renew-proxy.timer
 
+########################################
+%package -n xcache-consistency-check
+Summary: Consistency check for root files
+
+Requires: xz
+Requires: python-pip
+Requires: xrootd-server
```

Comment:
Since you're using logrotate, we should add that requirement here.

Reply:
And drop the python-pip requirement.

Reply:
since logs are going now to /var/log/xrootd/ and there is a logrotate for all log files in that folder, I will drop my logrotate script and won't add the dependency because that should be a dependency of xrootd-server which my script depends on